CHANGELOG.md

```diff
@@ -166,6 +168,38 @@
 
 #### Feat
 
+- New method to render template variables - `get_template_data()`
```

Comment:
Not related to this PR, but just realized that `get_template_data()` was not mentioned on its own